python/spacewalk/common/repo.py

```diff
@@ -426,12 +427,11 @@ def _get_release_index_from_file(self) -> typing.Dict[str, "DpkgRepo.ReleaseEntr
         # Repo format is not flat
         if not self.is_flat():
             if self.gpg_verify and not self._has_valid_gpg_signature(local_path):
-                # pylint: disable-next=logging-format-interpolation,consider-using-f-string
-                logger.error("GPG verification failed: {}".format(release_file))
+                # pylint: disable-next=logging-fstring-interpolation
+                logger.error(f"{DpkgRepo.GPG_VERIFICATION_FAILED}: {release_file}")
```

Comment:
Can we fix this completely?

Reply:
This should be equivalent and should pass the linting error (f-strings are eagerly evaluated, which is why they are discouraged in logging, since logging may never fire)

```suggestion
                logger.error("%s: %s", DpkgRepo.GPG_VERIFICATION_FAILED, release_file)
```